astroid/tests/unittest_inference.py

```diff
@@ -3336,6 +3336,20 @@ def blurb(self):
         self.assertIsInstance(inferred, nodes.Const)
         self.assertEqual(inferred.value, 25)
 
+    @test_utils.require_version('3.5')
+    def test_typing_namedtuple_is_callable(self):
+        import typing
+        node = extract_node("""
+        import typing
+        MyNamedTuple = typing.NamedTuple('Named', [('foo', int), ('bar', int)])
+        MyNamedTuple
+        """)
+        inferred = list(node.infer())
+        if util.Uninferable in inferred:
```

Comment:
I don't like it either. Does this happen with older versions of ``typing``? We should, probably, pin it to a version that doesn't give us too much trouble.